apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/SqlStore.java

```diff
@@ -0,0 +1,96 @@
+package com.apollographql.android.cache.normalized.sql;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteStatement;
+
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+
+import javax.annotation.Nullable;
+
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_KEY;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_RECORD;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.TABLE_RECORDS;
+
+final class SqlStore extends CacheStore {
```

Comment:
One thing to note -- I made CacheStore an abstract class with `load(Collection<Record>)` naively call `load(Record)`. 

I need to brush up on my sqlite, but there may be some performance benefits towards either writing a custom load multiple query, and/or using transactions.